packages/core/build/webpack.config.js

```diff
@@ -3,12 +3,14 @@ const { ALIASES, IS_RELEASE, MINIMIZERS, plugins, rules } = require('./constants
 const { openChromeBasedOnPlatform } = require('./helpers');
 
 module.exports = function(env, argv) {
-    const base = env && env.base && env.base != true ? '/' + env.base + '/' : '/';
+    const base = env && env.base && env.base !== true ? '/' + env.base + '/' : '/';
+    const sub_path = env && env.package && env.package !== true ? env.package : '';
```

Comment:
Should we change as follows?
```suggestion
    const sub_path = env && env.open && env.open !== true ? env.open : '';
```